.travis.yml

```diff
@@ -9,6 +9,7 @@ rvm:
 - 2.5.1
 
 before_install:
+  - git fetch --tags
```

Comment:
If you fetch tags here do you have to merge the tagged commit into master?

Reply:
Semantic release should take care of that.

Reply:
I say this because in .net and unity semantic release puts the tagged commit as an orphan off master and never merges it.

Reply:
That can possibly be the reason why builds are failing. The reason I added it is because I tried `git branch --contains v0.11.0` and it failed, I fetched tags and then I was able to see the tag.